crystal/grip/config.yaml

```diff
@@ -0,0 +1,4 @@
+framework:
+  website: https://github.com/grkek/grip
```

Comment:
rather use `github: grkek/grip` as shortcut

Reply:
I'll let you do it, since I am very new to this repo I don't want to mess anything up